Library/Homebrew/extend/os/mac/keg_relocate.rb

```diff
@@ -111,6 +123,12 @@ def each_install_name_for(file, &block)
     dylibs.each(&block)
   end
 
+  def each_rpath_for(file, &block)
+    rpaths = file.rpaths
+    rpaths.reject! { |fn| fn =~ /^@(loader|executable)_path/ }
```

Comment:
```suggestion
    rpaths = file.rpaths
                 .reject { |fn| fn =~ /^@(loader|executable)_path/ }
```

Reply:
These style comments (including your other one) also apply to identical code just above these lines, though. Should I apply those changes there as well?

I suppose I could also refactor the duplication into a separate method. Hasn't passed the rule-of-three bar for that though.

Reply:
> These style comments (including your other one) also apply to identical code just above these lines, though. Should I apply those changes there as well?

Yes, ideally!



> I suppose I could also refactor the duplication into a separate method. Hasn't passed the rule-of-three bar for that though.

May be worth it regardless, though?

Reply:
Cleaning up the other methods would make rebasing #11286 a bit messy, so I'll just work on them after that is merged. Left a note for myself: https://github.com/Homebrew/brew/pull/11286#issuecomment-833680029